spring-cloud-starter-stream-processor-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorflowProcessorProperties.java

```diff
@@ -37,24 +39,46 @@
 	private Resource modelLocation;
```

Comment:
ok i will change `modelLocation` to `model` and `mdoelFetchName` into `modelFetch`. So now we will have `model` (for the location), `modelFetch` for the model operation to fetch the output from and `modelFetchIndex` for the index of the `Tensor` from the modelFetch list.
✅